features/manage_mystuff.feature

```diff
@@ -1,12 +0,0 @@
-Feature: Manage My Stuff
```

Comment:
Isn't this coming back in some format?

Guess it doesn't matter, it'd be different enough they'd need to be totally re-written.